### PR TITLE
[OV JS] npm run lint on .js files

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - include *.js files
 - Fix codestyle errors about max-len, snake-case etc.

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/bindings/js/.eslintrc.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     './.eslintrc-global.js',
   ],
-  ignorePatterns: ['**/*.js', 'node_modules/', 'types/', 'dist/', 'bin/'],
+  ignorePatterns: ['node_modules/', 'types/', 'dist/', 'bin/'],
```
Comment (vishniakov-nikolai) [inline src/bindings/js/.eslintrc.js:7]:
```suggestion
  ignorePatterns: ['node_modules/', 'types/', 'dist/', 'bin/', '.eslintrc.js'],
```

**Review by vishniakov-nikolai — COMMENTED** [review]
Add at the top of `src\bindings\js\node\tests\e2e\electron-app.test.js`: 
`/* global describe, it, before, after */`

And content of `src\bindings\js\node\.eslintrc.js` should be:
```
module.exports = {
  extends: ['../.eslintrc.js'],
  env: {
    node: true,
  },
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],
};
```

After this PR merge I propose consider about this:
https://eslint.org/docs/latest/use/migrate-to-9.0.0

File: src/bindings/js/.eslintrc.js (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/.eslintrc.js:7]: Applied

**Review by vishniakov-nikolai — APPROVED** [review]
Cool

**Pull request merged**